glacier/src/exporters/svg.ts

```diff
@@ -0,0 +1,47 @@
+import * as vl from "vega-lite";
+declare global {
+    interface Element {}
+}
+import * as vega from "vega";
+import redux = require("redux");
+import {ModelState} from "../";
+import {Exporter} from "./";
+
+export function createSvgExporter(store: redux.Store<ModelState>) {
```

Comment:
instead of hard-coding values maybe spec should be a parameter of this function?